test/functional/test_framework/blocktools.py

```diff
@@ -110,6 +112,16 @@ def create_transaction(prevtx, n, sig, value, scriptPubKey=CScript()):
     tx.calc_sha256()
     return tx
 
+
+def create_coinbase_with_bad_txin(*args, **kwargs):
+    """Return a coinbase CTransaction with an invalid txin."""
+    kwargs['txin'] = CTxIn(
```

Comment:
In commit "[tests] Add utility for generating bad coinbase transactions"

Could drop kwargs assignment with

```python
return create_coinbase(*args, **kwargs. txin=CTxIn(...))
```

Reply:
Ah, Python 3.4 doesn't support this syntax (which is what we run on Travis). Will have to revert this.